MariaDB/server
# MDEV-27965: MariaDB needs at least libfmt version 7.0

- [x] *The Jira issue number for this PR is: MDEV-27965*

## Description
Add requirement in Debian control file for libfmt 7.0 which is needed for building MariaDB

## How can this PR be tested?
Try to build package without libfmt-dev >= 7.0.0 and it should fail

See Salsa-CI run:
https://salsa.debian.org/illuusio/mariadb-server/-/pipelines/360727

## Basing the PR against the correct MariaDB version

## Backward compatibility
As you need newer libfmt there is needed attention in `autobake.sh`.
